controllers/pvc-mutator/storage-class-label-sync/sync.go

```diff
@@ -0,0 +1,55 @@
+package storageclasslabelsync
```

Comment:
This is a bit of a mouthful - how about `sclabel`?  This matches better with existing packages: `pvclabel`, `nodelabel`, `nodedelete` and `nsdelete`.

Same with the directory name, I think `sc-label` would be better.